napari/_qt/layer_controls/qt_vectors_controls.py

```diff
@@ -55,15 +55,14 @@ class QtVectorsControls(QtLayerControls):
     def __init__(self, layer) -> None:
         super().__init__(layer)
 
-        # dropdown to select the property for mapping edge_color
-        color_properties = self._get_property_values()
+        # dropdown to select the feature for mapping edge_color
         self.color_prop_box = QComboBox(self)
         self.color_prop_box.currentTextChanged.connect(
             self.change_edge_color_property
         )
-        self.color_prop_box.addItems(color_properties)
+        self.color_prop_box.addItems(self.layer.features.columns)
```

Comment:
Considered renaming the names of these attributes from `prop` to `feature`, but I think this technically part of the napari package public API. I.e. I think someone can access it with something like:

```python
vectors = viewer.add_vectors()
qt_viewer = QtViewer(viewer)
qt_viewer.controls.widgets[vectors].color_prop_box
```

though I'm not sure if that's common or expected.

Reply:
hmm good point. I'm not sure the best thing to do here. It feels okay to leave as-is, but I'm not sure.

Reply:
I'm going to leave them alone for now. I don't think the variable names here are critical and I don't want to break anyone unnecessarily.